### PR TITLE
Remove penalty for amphibious and aquatic unarmed strikes underwater

### PR DESCRIPTION
**Pull request by TikaelSol**
They don't take those penalties on their unarmed strikes per their traits.

### CODE REVIEW

**Comment by stwlam** [comment]
This would be limited to NPCs

**Comment by TikaelSol** [comment]
I'm not sure that's the case actually. You don't gain the benefits of actor traits from your ancestry, like construct, but this isn't a benefit of the trait itself, it's a rules interaction.

**Comment by stwlam** [comment]
I would agree except the benefit comes from the trait description itself

File: packs/other-effects/effect-aquatic-combat.json [hunk 1]
```diff
@@ -32,6 +32,18 @@
...
+                    },
+                    {
+                        "nand": [
+                            {
+                                "or": [
+                                    "self:trait:amphibious",
+                                    "self:trait:aquatic"
+                                ]
+                            },
+                            "unarmed",
```
Comment (stwlam) [inline packs/other-effects/effect-aquatic-combat.json:44]:
```suggestion
                            "item:category:unarmed",
```

**Pull request merged**